File: materialpreference/src/main/java/com/anggrayudi/materialpreference/StoragePermissionCallback.java
package com.anggrayudi.materialpreference;

public interface StoragePermissionCallback {

    /**
     * Called when some permissions are missing.
     * @param read <code>true</code> if granted
     * @param write <code>true</code> if granted
     */
    void onPermissionTrouble(boolean read, boolean write);
}

File: materialpreference/src/main/java/com/anggrayudi/materialpreference/callback/StoragePermissionCallback.kt
package com.anggrayudi.materialpreference.callback

interface StoragePermissionCallback {

    /**
     * Called when some permissions are missing.
     * @param read `true` if granted
     * @param write `true` if granted
     */
    fun onPermissionTrouble(read: Boolean, write: Boolean)
}
